### ISSUE TITLE
Can’t access `stdout` when watching directory

### ISSUE THREAD

**Issue by niksy** [sass/node-sass#1353]
I’m running CLI process from JS file with following commands:

```js
var exec = require('execa');

var child = exec.spawn('node-sass', ['input', '--watch', '--output', 'output']);

child.stdout.on('data', function ( data ) {
	console.log(data.toString());
});
```

Everything compiles successfuly, but I can’t access `stdout` data via event listener. When running same command inside shell, everything is OK.

I’ve tried to access `EventEmitter` events returned from process, but also nothing happens.

```js
child.on('warn', function ( data ) {
	console.log(data);
});
```

`node-sass` is installed locally. Running commands such as getting module version returns proper 'data' event on `stdout`.

Am I doing something wrong?

**Comment by niksy** [comment]
OK, I’ve managed to get information on `stderr` stream regardless of error or success. I suppose that’s error?

**Comment by JeffreyWay** [comment]
Yep, getting the same here. Been over a year. Any chance of this being fixed?
